packages/server/lib/plugins/util.js

```diff
@@ -4,11 +4,64 @@ const debug = require('debug')('cypress:server:plugins')
 const Promise = require('bluebird')
 
 const UNDEFINED_SERIALIZED = '__cypress_undefined__'
+const FUNCTION_SERIALIZED = '__cypress_function__'
 
 const serializeError = (err) => {
   return _.pick(err, 'name', 'message', 'stack', 'code', 'annotated', 'type')
 }
 
+function serializeArgument (arg) {
```

Comment:
This function might be a good candidate for some unit tests - not sure how thorough it is covered by the e2e tests.